### PR TITLE
Feat/new layout

### PR DESCRIPTION
**Pull request by GoOz**
I added the new layout to allow mobile views to use minimal-ui.

Due to this change, some components had to be updated:
- [x] `<UploadQueue />` is positioned over the content instead of pushing it. (OK with Design team)
- [x] Refactored `.fil-content-file` from tables so that whatever the number of lines (filename and/or path) the height of this element stays the same.
- [x] Removed `<AutoSizer />` from `react-virtualized` and used `react-measure` instead to set the width of PhotoList because `<AutoSizer />` was using a layout method incompatible with the new layout.

~~⚠️ Needs a new Cozy-UI release~~ **UI released**

### CODE REVIEW

**Comment by GoOz** [comment]
[(2) 🐛 Problème de scroll sur safari mobile](https://trello.com/c/hO0d4QBy/463-2-%F0%9F%90%9B-probl%C3%A8me-de-scroll-sur-safari-mobile)

**Comment by y-lohse** [comment]
Related to https://github.com/cozy/cozy-ui/pull/247

File: src/drive/ducks/upload/styles.styl [hunk 1]
```diff
@@ -158,16 +158,19 @@ progress.upload-queue-progress::-moz-progress-bar
     $coz-bar-border-size = 1px
 
     .upload-queue
-        position static
+        top $coz-bar-size
```
Comment (y-lohse) [inline src/drive/ducks/upload/styles.styl]: There's no position anymore ?

File: src/photos/ducks/upload/styles.styl [hunk 2]
```diff
@@ -156,16 +156,19 @@ progress.upload-queue-progress::-moz-progress-bar
     $coz-bar-border-size = 1px
 
     .upload-queue
-        position static
+        top $coz-bar-size
```
Comment (y-lohse) [inline src/photos/ducks/upload/styles.styl]: Same thing, no position ?

File: src/drive/ducks/upload/styles.styl (hunk 1 above)
Comment (GoOz) [reply src/drive/ducks/upload/styles.styl]: there is but it keeps the fixed position set for desktop
https://github.com/GoOz/cozy-drive/blob/603682218a6fc3d18963c5bc406fe562b4283093/src/drive/ducks/upload/styles.styl#L6

File: src/photos/ducks/upload/styles.styl (hunk 2 above)
Comment (GoOz) [reply src/photos/ducks/upload/styles.styl]: same answer :)

**Pull request merged**